posts/2020-06-03-asynchronous-programming-made-resilient-with-mpft.adoc

```diff
@@ -30,7 +30,7 @@ But there's one, in particular, that's going to help you make your asynchronous
 Adding the `@Asynchronous` annotation to a method does two things:
 
 1. Runs the method asynchronously
-2. Allows Fault Tolerance strategies to be applied to asynchronous methods
+2. If the method returns a `CompletionStage`, applies other fault tolerance strategies to the result of the `CompletionStage`, rather than the result of the method.
```

Comment:
Suggestion

> If the method returns a `CompletionStage`, applies other Fault Tolerance strategies to the result of the `CompletionStage`, rather than to the result of the method